File: src/test/scala/cern/colt/matrix/tint/impl/DenseIntMatrix2DViewTest.scala
package cern.colt.matrix.tint.impl

//remove if not needed
import scala.collection.JavaConversions._

class DenseIntMatrix2DViewTest(arg0: String) extends DenseIntMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new DenseIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new DenseIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new DenseIntMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tint/impl/DenseIntMatrix2DViewTest.java
package cernx.colt.matrix.tint.impl;

public class DenseIntMatrix2DViewTest extends DenseIntMatrix2DTest {

    public DenseIntMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseIntMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
